lua/lualine/extensions/nvim-dap-ui.lua

```diff
@@ -0,0 +1,13 @@
+-- MIT license, see LICENSE for more details.
+-- Extension for nvim-dap-ui
+local M = {}
+
+M.sections = {
+  lualine_a = { 'filename' }
```

Comment:
Set `file_status` option to false then [-] won't appear . Also it seems after last commit there are merge conflicts.

Reply:
ok thanks for the suggestion, i added the file_status option